DataWorkerService/Services/Implementations/PlayersService.cs

```diff
@@ -145,7 +145,7 @@ public async Task UpdateOutdatedFromOsuTrackApiAsync(PlayerFetchPlatformConfigur
             await UpdateFromOsuTrackApiAsync(player);
         }
 
-        await context.SaveChangesAsync();
+        await playersRepository.UpdateAsync(outdatedPlayers);
```

Comment:
Just thought id mention that the `repository.UpdateAsync()` technically doesn't do anything different than `context.SaveChangesAsync()` when the change tracker is enabled. In fact because of a quirk with `dbSet<T>.UpdateAsync()` it actually makes EF think that _all_ fields of an entity require updating and will execute a replace on all columns for that row. Weird stuff, but that is why in the data worker I preferred saving directly from the context

Reply:
I mainly called it this way to have consistent update behavior. It would feel odd to me to not call UpdateAsync because then not all of the entity's update behavior is happening inside the repository methods. Makes for some head scratching later on if we need to debug why something isn't updating as expected.